src/builder.rs

```diff
@@ -47,6 +47,11 @@ impl<K: EnrKey> EnrBuilder<K> {
         self.add_value_rlp(key, rlp::encode(&value).freeze())
     }
 
+    // TODO @age: Should this fn be pub?
+    fn add_integer<T: Encodable>(&mut self, key: impl AsRef<[u8]>, value: &T) -> &mut Self {
```

Comment:
I think we actually probably want to replace `add_value` with this function. 

The `add_value` seems like it promotes the wrong behaviour. 

Also then all the following renaming changes are no longer necessary. 

It will change the API, but probably for the better